index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>portofolio: setup and mobile version skeleton</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body class="">
+    <header class="navibar"> 
+        <nav class="nav-container">
+<a class="logo" href="#">My Logo</a>
+<img class="mobile-menu" src="img/Union.png" alt=" navigation">
+
+</nav>
+    </header>
```

Comment:
- [ ] Please fix indentation spaces lines 13, 14, and 16 and kindly remove empty line 15 to make your code more readable and clean.